meeting-notes/2019/Q1/2019-01-07--go-core-dev-team-weekly.md

```diff
@@ -0,0 +1,183 @@
+# 💫Golang Core Dev Team Weekly Sync 🙌🏽 January 7, 2019
+
+- **Lead:** @eingenito
+- **Notetaker:**
+- **Attendees:**
+  - _attendee names..._
+  - @mgoelzer
+  - @aschmahmann
+  - @momack2
+  - @stebalien
+  - @michaelavila
+  - @daviddias
+  - @bigs
+  - @djdv
+  - @kevina
+  - @warpfork
+- **Recording:** _add link to recording once it's online_
+
+## Agenda
+
+- Ask everyone to put their name into the list of attendees
+- Round of updates
+  - What have you accomplished since the last Weekly?
+  - Were there any blockers? If so, which ones? Is it still blocked? Why?
+  - What is the next important thing you should focus on?
+  - How much fun did you have working on ipfs/ipld/libp2p last week?
+- Ask for general questions. Could be things like:
+  - I'm stuck with something, I don't know who to ask. Who knows who to ask?
+  - Who can help me with xyz?
+- Plan this week
+  - Is anybody looking for work?
+  - Are the high priority issues that need attention? 
+- New meeting format proposal (https://github.com/ipfs/team-mgmt/issues/827) - @stebalien
+  - Meeting:
+    1. Blockers/asks (no updates)
+    2. Announcements
+    3. Agenda
+  - Async:
+    - Every large endeavor gets a meta issue ("epic issue").
+    - Updates are posted in these issues.
+- Gateway Issue - @stebalien
+  - Believed to be https://github.com/ipfs/go-bitswap/issues/53
+    - We're not connecting to providers.
+    - Deeper issue: we should be reconnecting to peers in a session.
```

Comment:
Note to reader: there are more complex issues than the ones that can be solved by a hotfix. The "Fix the Gateways" meeting was productive, Now we have some really good ideas anda tiger team / task force spearheaded by @eefahy